megamek/i18n/megamek/client/messages.properties

```diff
@@ -39,12 +39,25 @@ SkillGeneratorType.MANEI_DOMINI.toolTipText=Generate skills for Manei Domini cre
 
 
 ##### UI Resources
+#### Base Components
+#### AbstractMHQNagDialog
+chkIgnore.text=Do not bother me again
+chkIgnore.toolTipText=If selected this nag will no longer show until re-enabled under MekHQ Options.
```

Comment:
Is this MekHQ specific?

Reply:
Dumb copy-paste error, the dialog was created based on MHQ's.